File: examples/src/Presentations/Conversion/ODPToPPTX.cs
import aspose.slides as slides
using Aspose.slides.Examples.CSharp
using Aspose.slides.Export
using System
using System.Collections.Generic
using System.Linq
using System.text
using System.Threading.Tasks

namespace CSharp.Presentations.Conversion
{
    class ODPToPPTX
    {

        public static void Run() {

            #ExStart:ODPToPPTX

            # The path to the documents directory.
            dataDir = RunExamples.GetDataDir_Conversion()

            
            srcFileName = dataDir + "AccessOpenDoc.odp"
            destFileName = dataDir + "AccessOpenDoc.pptx"
            #Instantiate a Presentation object that represents a presentation file
            using (Presentation pres = new Presentation(srcFileName))
            {
                #Saving the PPTX presentation to PPTX format
                pres.save(destFileName, slides.export.SaveFormat.PPTX)
            }
            #ExEnd:ODPToPPTX



        }
    }
}

File: examples/src/Presentations/Conversion/ODPToPPTX.py
import aspose.slides as slides

def convert_to_odp():
    dataDir = "./examples/data/"
    outDir = "./examples/out/"

    with slides.Presentation(dataDir + "welcome-to-powerpoint.pptx") as pres:
        pres.save(outDir + "convert_to_odp_out.odp", slides.export.SaveFormat.ODP)

    with slides.Presentation(outDir + "convert_to_odp_out.odp") as pres:
        pres.save(outDir + "convert_to_odp_out.pptx", slides.export.SaveFormat.PPTX)